### ISSUE TITLE
Remove past year code that won't be used this year

### ISSUE THREAD

**Issue by akouete-kpakpo** [ML-for-B-E/Optimisation#4]
## Tasks

- [ ] remove from the notebook the part on Neural Network
- [ ] remove from the src code the part that won't be roll forward

**Comment by rleriche** [comment]
On ne garde pas la partie réseau de neurones dans le TP ? Juste l'optimisation de L1sphere en faisant varier lambda, la pénalisation L1 ? Si c'est ça l'idée, alors on peut effectivement couper la fin des notebooks. Sinon c'est mieux de les garder. Autre solution : le mettre en question complémentaire si certains sont motivés.

**Comment by BWhannou** [comment]
@akouete-kpakpo why do you want to remove the on Neural Network? I would like share it with the students. that's why they have a course on Neural Network before our Optimization course. Or am I missing something?

**Comment by akouete-kpakpo** [comment]
@rleriche we'll keep the neural network part

I don't remember exactly the context of this issue. I think we decided this after our meeting earlier in the year.

@BWhannou ?

**Comment by BWhannou** [comment]
Yes, @akouete-kpakpo you are right. Now I remember, we planned to change the neural network with the loss function of other models. But we didn't have the time. I propose to close it. And keep this as is. Maybe we will do another TP if necessary.
